tests/test_data/pinned.json

```diff
@@ -10,7 +10,7 @@
   "delegations": [
     {
       "paths": ["*"],
-      "repositories": ["mainrepo", "director"]
+      "repositories": ["imagerepo", "director"]
```

Comment:
Is there a reason why `imagerepo` is sometimes used, and `image_repo` at other times?